esinstall/src/entrypoints.ts

```diff
@@ -39,17 +48,11 @@ export function findManifestEntry(
     }
   }
 
-  foundEntrypoint = [
-    ...packageLookupFields,
-    'browser:module',
-    'module',
-    'main:esnext',
-    'jsnext:main',
-  ]
+  foundEntrypoint = [...packageLookupFields, ...MAIN_FIELDS]
     .map((e) => manifest[e])
     .find(Boolean);
 
-  if (foundEntrypoint) {
```

Comment:
This additional check is required since `browser` is now included when searching for `foundEntrypoint`.